fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -255,7 +255,7 @@ charsetName
     ;
 
 defaultDesc
-    : DEFAULT (string| NULL | CURRENT_TIMESTAMP)
+    : DEFAULT (string | NULL | CURRENT_TIMESTAMP | '(' qualifiedName '(' ')' ')')
```

Comment:
Why not use expr?

Reply:
DEFAULT <expr>
snowflake does not have parentheses.

Reply:
> Why not use expr?

because expr is more complicated so far, just want to support some parameterless functions for Now.

Reply:
> DEFAULT <expr> snowflake does not have parentheses.

mysql syntax has parentheses, we should first compatible with mysql syntax.